src/equisolve/numpy/models/__init__.py

```diff
@@ -0,0 +1,11 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+#
+# Copyright (c) 2022 Authors and contributors
+# (see the AUTHORS.rst file for the full list of names)
+#
+# Released under the GNU Public Licence, v3 or any higher version
+# SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
What? The repo says MIT, and the rest of the ecosystem is BSD =)

Reply:
Let me change this to BSD if this is our license

Reply:
I don't think we need to include this boilerplate in every files, but I would be in favor of changing the licence of this repo to BSD. This should be done in a separate PR though!